frontend/public/style/_common.scss

```
@@ -571,6 +571,11 @@
   }
 }
 
+.co-storage-default {
```

```suggestion
.co-storage-class-default {
```

Since this is specific to the storage class page, it should be in a separate `_storage-class.scss` file rather than `_common.scss`. Make sure to add it to `style.scss`.

---

Ack!